src/head.rs

```diff
@@ -0,0 +1,80 @@
+use bevy::{math::DVec2, prelude::*, utils::Uuid};
+
+use bevy_vello::prelude::*;
+
+#[derive(Default, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct ShapeId(Uuid);
+
+#[derive(Component, Copy, Clone)]
+pub struct Head {
+    pub shape_id: ShapeId,
+    pub time: f64,
+
+    pub scale: f64,
+    pub offset: f32,
+    pub rotation_offset: f32,
+}
+
+impl Default for Head {
+    fn default() -> Self {
+        Self {
+            shape_id: ShapeId(Uuid::new_v4()),
+            time: 1.0,
+            scale: 1.0,
+            offset: 0.0,
+            rotation_offset: 0.0,
```

Comment:
`Self {time: 1.0, /* ... */, ..default()}` crashes here for some reason? so i have to do it manually.